R/textanal.R

```diff
@@ -566,7 +566,13 @@ exp_topic_model <- function(df, text,
 #' @export
 #' @param type - Type of output.
 tidy.textmodel_lda_exploratory <- function(x, type = "doc_topics", num_top_words = 10, ...) {
-  if (type == "word_topics") {
+  if (type == "topics_summary") { # Count number of documents that "belongs to" each topic.
+    docs_topics_df <- as.data.frame(x$model$theta)
+    docs_topics_df <- docs_topics_df %>% dplyr::mutate(topic = summarize_row(across(starts_with("topic")), which.max))
+    res <- docs_topics_df %>% dplyr::select(topic) %>% dplyr::group_by(topic) %>% dplyr::summarize(n=n())
+    res <- res %>% tidyr::complete(topic = 1:x$model$k, fill = list(n=0)) # In case some topic do not have any doc that "belongs to" it.
```

Comment:
Please add a test to check that the missing values are completed and 0s are filled.